### PR TITLE
Fix build error

### PR DESCRIPTION
**Pull request by cydrain**
Issue: https://github.com/milvus-io/milvus/issues/38631

### CODE REVIEW

File: cmake/libs/libfaiss.cmake [hunk 1]
```diff
@@ -73,7 +74,6 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Android" AND CMAKE_SYSTEM_PROCESSOR STREQUAL "aar
   find_package(OpenBLAS REQUIRED)
   set(BLAS_LIBRARIES OpenBLAS::OpenBLAS)
 else()
-  find_package(LAPACK REQUIRED)
```
Comment (foxspy) [inline cmake/libs/libfaiss.cmake:76]: @Presburger pls help take a look~

**Comment by alexanderguzhva** [comment]
lgtm overall
but why was it compiling before?

**Comment by cydrain** [comment]
> lgtm overall but why was it compiling before?

I can't explain clearly why the change of this line caused Milvus build fail. I encountered this compilation issue in my desktop, but it seems that others are compiling normally.
This issue was introduced in PR # 959, and I fixed a part of it in PR # 979. This PR changed the CMakeLists.txt back to its original state before PR # 959

**Comment by alexanderguzhva** [comment]
> > lgtm overall but why was it compiling before?
> 
> I can't explain clearly why the change of this line caused Milvus build fail. I encountered this compilation issue in my desktop, but it seems that others can compile successfully with same code. This issue was introduced in PR # 959, and I fixed a part of it in PR # 979. This PR changed the CMakeLists.txt back to its original state before PR # 959

Would it be possible to verify that the Android compiltation does not break?

**Comment by cydrain** [comment]
> > > lgtm overall but why was it compiling before?
> > 
> > 
> > I can't explain clearly why the change of this line caused Milvus build fail. I encountered this compilation issue in my desktop, but it seems that others can compile successfully with same code. This issue was introduced in PR # 959, and I fixed a part of it in PR # 979. This PR changed the CMakeLists.txt back to its original state before PR # 959
> 
> Would it be possible to verify that the Android compiltation does not break?

Discussed with @Presburger , it should be my env issue, Milvus can build pass in the docker env.
So close this PR.

**Pull request closed without merging**